src/views/v2/tickets/item.php

```diff
@@ -66,18 +66,21 @@
 	<?php endif; ?>
 >
 
-	<?php $this->template( 'v2/modal/item/remove', $context ); ?>
+	// @todo Convert this into an action.
+	<?php //$this->template( 'v2/modal/item/remove', $context ); ?>
 
 	<?php $this->template( 'v2/tickets/item/content', $context ); ?>
 
 	<?php $this->template( 'v2/tickets/item/quantity', $context ); ?>
 
 	<?php $this->template( 'v2/tickets/item/quantity-mini', $context ); ?>
 
-	<?php $this->template( 'v2/modal/item/total', $context ); ?>
+	// @todo Convert this into an action.
+	<?php //$this->template( 'v2/modal/item/total', $context ); ?>
```

Comment:
* :warning: This comment is 67% valid code; is this commented out code?
* :no_entry_sign: No space found before comment text; expected "// $this->template( 'v2/modal/item/total', $context );" but found "//$this->template( 'v2/modal/item/total', $context );"

<sub>hash: 35ad4d3179a4646d411e67b97f0462f9</sub>